tests/estimators/speech_recognition/test_tensorflow_lingvo.py

```diff
@@ -0,0 +1,329 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from numpy.testing import assert_allclose, assert_array_equal
+from scipy.io.wavfile import read
+
+from art.estimators.speech_recognition.speech_recognizer import SpeechRecognizerMixin
+from art.estimators.speech_recognition.tensorflow_lingvo import LingvoAsr
+from art.estimators.tensorflow import TensorFlowV2Estimator
+from art.utils import get_file
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
```

Comment:
1/ So one of the conventions we are using in the new tests is to place any new fixtures within `conftest.py` files in the relevant package. This is to make sure we reuse as much as possible fixtures across test scripts to avoid duplication of code. So in this case both `audio_data` and `audio_batch_padded` fixtures should be moved to a new `conftest.py` file located at `tests/estimators/speech_recognition/`. Please see See section 3.2 in [wiki doc](https://github.com/Trusted-AI/adversarial-robustness-toolbox/wiki/ART-Unit-Testing) about creating fixtures. 

Also, I guess we can move this to Slack but I'll also make sure that these fixtures are added to the wiki test documentation so that other people can reuse that fixture in the future if needed.